js/src/admin/utils/getCategorizedExtensions.js

```diff
@@ -0,0 +1,29 @@
+export default function getCategorizedExtensions() {
+  let extensions = {};
+
+  Object.keys(app.data.extensions).map((id) => {
+    const extension = app.data.extensions[id];
+    let category = extension.extra['flarum-extension'].category;
+
+    if (!extension.extra['flarum-extension'].category) {
+      category = 'other';
+    }
+
+    // Wrap languages packs into new system
```

Comment:
This should be clarified to explain what "new system" means. "All language packs should be classified as such, regardless of their direct classification"?

Reply:
That is for language packs that don't get updated with the category field. I'll add an extra check.